src/module-declaration.ts

```diff
@@ -44,7 +44,11 @@ export const generateModuleDeclaration = (
           `${isClass ? 'class' : 'interface'} ${_.upperFirst(
             module.name,
           )} extends ${module.extends ||
-            (module.name === 'remote' ? 'MainInterface' : 'NodeJS.EventEmitter')} {`,
+            (module.name === 'remote'
+              ? 'MainInterface'
+              : isClass
+              ? 'EventEmitter'
```

Comment:
```suggestion
              ? 'NodeEventEmitter'
```

To match above suggestion